webapp/packages/plugin-data-viewer-result-trace-details/src/DVResultTraceDetailsPresentation.tsx

```diff
@@ -52,22 +27,54 @@ export const DVResultTraceDetailsPresentation: DataPresentationComponent = obser
   const state = useResultTraceDetails(model, resultIndex);
 
   useAutoLoad(DVResultTraceDetailsPresentation, state, undefined, undefined, true);
+  const trace = state.trace;
+
+  const columnsCount = useCreateGridReactiveValue(() => 3, null, []);
+  const rowCount = useCreateGridReactiveValue(
+    () => trace?.length || 0,
+    onValueChange => reaction(() => trace?.length || 0, onValueChange),
+    [trace],
+  );
+
+  function getCell(rowIdx: number, colIdx: number) {
+    switch (colIdx) {
+      case 0:
+        return trace![rowIdx]?.name ?? '';
+      case 1:
+        return trace![rowIdx]?.value ?? '';
+      case 2:
+        return trace![rowIdx]?.description ?? '';
```

Comment:
why do we need this `trace` variable and why do we access it with `trace?` and `trace!` at the same time?